GDJS/Runtime/inputmanager.ts

```diff
@@ -309,16 +326,18 @@ namespace gdjs {
     }
 
```

Comment:
I hesitated to do it. I though is was too verbose and could actually make the code less readable as all the `+ 1` all are in the same location and it is hard to find a function name that is understandable without documentation, but I'm ok to make the change. It would be an "external" identifier as it's the code used by events.

Reply:
> A local variable would probably be clearer.

Unless the variable name is very verbose, understanding it would require the comment to be above each local variable, which is not awesome (especially for the one liner in `onTouchMove`). I think a function is cleaner because it allows the explanation comment to only be used once, and when editing the other functions you do not need to know why an identifier needs 1 added to it. Ensuring an ID is not 0 is another concern than adding, modifying and deleting a touch, and should therefore be separated in my opinion

> It would be an "external" identifier as it's the identifier used by events.

Fair, I'd suggest naming it `publicID` then.

> I though is was too verbose and could actually make the code less readable

In that case we should probably ask for a 3rd opinion

Reply:
> It would be an "external" identifier as it's the identifier used by events.

That's fair! It's indeed a public/external identifier.

> > I though is was too verbose and could actually make the code less readable
> In that case we should probably ask for a 3rd opinion

My decision would be to go with the more verbose alternative suggested by Arthuro555: a function to "do the +1".
Sure it's not a lot, but:
- it's not performance sensitive code (we don't care about an extra function call).
- it's more verbose, but it also force you to think "am I right to translate this raw identifier into a public identifier?" and "because it's verbose, I'll try to use it only once" 
  - We could almost think of two separate types (but no need to do this, not sure we can express this in TypeScript anyway).
- Finally, it's a good place to put a comment.